jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -166,7 +166,7 @@ preferences.start_jobs=Autom. Hintergrunddekompilierung starten
 preferences.select_font=Ändern
 preferences.select_smali_font=Ändern
 preferences.deobfuscation_on=Deobfuskierung aktivieren
-#preferences.deobfuscation_map_file_mode=Map file handle mode
+preferences.deobfuscation_map_file_mode=Datei-Handle-Modus zuordnen
```

Comment:
This preference entry is about how Jadx handles the deobfuscation map files (values: `READ`, `READ_OR_SAVE`, `OVERWRITE`, `IGNORE`), so `Datei-Handle-Modus zuordnen` does not really fit here in my opinion. But I also don't have a good translation.

Reply:
@Henry2o1o What do you think about `Map-Datei Modus`?

Reply:
https://github.com/skylot/jadx/issues/1513

🤷‍♂️

Reply:
@Henry2o1o Not sure what your comment means (sorry but I don't speak "smiley"). #1513 is about the removal of the preference `deobfuscationForceSave`.

Reply:
Umgang mit Map-Dateien
Handhabung der Map-Dateien